README.md

```diff
@@ -1 +1,51 @@
-# typewriter
\ No newline at end of file
+# Typewriter
+
```

Comment:
I think having a travis-ci badge here would be pretty cool, what do you think?

Reply:
Done!